src/main/java/com/amazon/opendistroforelasticsearch/security/OpenDistroSecurityPlugin.java

```diff
@@ -207,28 +210,40 @@ private static boolean isDisabled(final Settings settings) {
     private static boolean isSslOnlyMode(final Settings settings) {
         return settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_SSL_ONLY, false);
     }
-    
+
+    /**
+     * SSL Cert will be reloadable if security is enabled and we are not in sslOnly mode
+     * @param settings Elastic configuration settings
+     * @return true if ssl cert reload is enabled else false
+     */
+    private static boolean isSslCertReloadable(final Settings settings) {
+        return !isDisabled(settings) && !isSslOnlyMode(settings) && settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_SSL_CERT_RELOAD, false);
```

Comment:
Users should have the option of disabling hot reload, esp if they have their own on prem systems where they do not want to use this feature since it involves dynamically picking up sensitive certificate info from their nodes.

Reply:
This would imply that they would have certificates that never expire (long expiry) or would need a cluster restart to change this setting (restarting would spoil the purpose of having this feature itself).
This feature can be used only by security-admin. Anyways we should ensure picking up certificate info from the nodes is done safely.

Reply:
@hardik-k-shah whats the rationale behind adding this flag in settings ?